packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -162,7 +162,7 @@ export default class ConfigParser {
      * merge external object with config object
      * @param  {Object} object  desired object to merge into the config object
      */
-    private merge(object: MergeConfig = {}) {
+    private merge(object: MergeConfig = {}, shouldSetPathToSpecs = true) {
```

Comment:
I think it might make more sense to rename this to `addPathToSpecs`, what do you think?

Reply:
It seemed to me that in this case the modal verb in the name more clearly conveys that this is a boolean flag. However, I don't see any problem changing the name to the one you suggested